src/containers/Permissions.jsx

```diff
@@ -50,19 +50,29 @@ const Permissions = () => {
           {queryResultApps.data
             .concat(queryResultKonnectors.data)
             .sort((a, b) => a.slug.localeCompare(b.slug))
-            .map(appOrKonnector => (
-              <div key={appOrKonnector.name}>
-                <Link to={'/permissions/' + appOrKonnector.slug}>
-                  <ListItem button>
-                    <ListItemIcon>
-                      <AppIcon app={appOrKonnector} />
-                    </ListItemIcon>
-                    <ListItemText primary={appOrKonnector.name} />
-                  </ListItem>
-                </Link>
-                <Divider />
-              </div>
-            ))}
+            .map(appOrKonnector => {
+              return (
+                <div key={appOrKonnector.name}>
+                  <Link to={'/permissions/' + appOrKonnector.slug}>
+                    <ListItem button>
+                      <ListItemIcon>
+                        <AppIcon app={appOrKonnector} />
+                      </ListItemIcon>
+                      <ListItemText
+                        primary={appOrKonnector.name}
+                        secondary={
+                          appOrKonnector.permissions
+                            ? Object.entries(appOrKonnector.permissions)
+                                .length + ' permissions'
+                            : '0' + ' permissions'
```

Comment:
it should be possible to handle the plural with transifex. (when there is one permission)

Reply:
Referring to https://github.com/cozy/cozy-drive/blob/master/src/drive/locales/en.json#L259

Reply:
We're using the node version of polyglot https://airbnb.io/polyglot.js/ but this is the same thing. You can use a `%{smart_count}` variable ;)